Update README.md (#20262)
DataDog/integrations-core: oracle/README.md

### Custom query

> Custom Query is used to get specific fields from the database to be used a custom metrics.

Reviewer: ```suggestion
Custom Query is used to extract specific fields from the database to be used as custom metrics.
```

| `columns`       | This is a list representing each column, ordered sequentially from left to right. There are two required pieces of data: <br> a. `type` - This is the submission method (`gauge`, `count`, etc.). <br> b. name - This is the suffix used to form the full metric name. If `type` is `tag`, this column is instead considered as a tag which is applied to every metric collected by this particular query. |

Optionally use the `tags` parameter to apply a list of tags to each metric collected.

> The **number of columns** Output from the Custom Query should be **equal to to number of metrics variables** in the conf.yaml file shown below.

Reviewer: ```suggestion
The **number of columns** output from the Custom Query should be equal to to number of metrics variables in the `conf.yaml` file shown below.
```
